### PR TITLE
Library UI

### PR DESCRIPTION
**Pull request by David-Lanzz**
# Hello Reviewer!!!👍
232341638-bede8487-52fa-49f1-82d6-f6e74207dd91

## I have completed all tasks assigned to me, and below is a summary of my work 👇
- Correct use of gitflow👍
- No linter errors👍
- Implementation of Library app UI👍
- Task completion👍

### CODE REVIEW

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @David-Lanzz 👋,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Status: Invalid Code Review Request



You have submitted a project, that has failed linter errors. Kindly fix them.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: methods/app.rb [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    puts 'Age?'
+    age = gets.chomp
+    puts 'Name?'
+    name = gets.chomp
+    puts 'Has parent permission[Y/N]?'
+    parent_permission_input = gets.chomp.upcase
+    return unless parent_permission_input == 'Y'
+
+    new_student = Student.new(name, age, parent_permission_input)
+    @people.push(new_student)
```
Comment (Zuhaib042) [inline methods/app.rb]: - [ ] In the video given in the requirements, it shows that even `parent permission = n`, a student gets created.
but in your app, a student only gets created when the parent_persmission = y.
kindly consider fixing this behaviour.

File: methods/app.rb [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    end
+  end
+
+  def new_book
+    puts 'Book title?'
+    title = gets.chomp
+    puts 'Book author?'
+    author = gets.chomp
+    newest_book = Book.new(title, author)
+    puts newest_book
```
Comment (Zuhaib042) [inline methods/app.rb]: - [ ] Kindly consider removing this line as there is no need to show a book object in its raw form `#<Book:0x00000001051f5968>`, as it gives a bad user experience.

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @David-Lanzz 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✅ Descriptive PR
✅ Professional Readme

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @David-Lanzz , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- You successfully completed all the requirements.  🎉 🎉 🎉 ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**